### ISSUE TITLE
Overview information to form more realistic expectations on card issuing 

### ISSUE THREAD

**Issue by sergfeldman** [ubiquity/research#79]
### Intro

Gift cards stream is one of the streams of the card issuing project.
Currently, majority of team members do not deeply understand the project that leads to non-realistic expectations.

There is an overview information about the gift cards stream.
The metrics and status are not final as the stream is in progress now.

The purpose of providing this information is other team members can form their expectations based on the real situation and complexity of the project.

More realistic expectations will help us proceed more effectively and achieve better results as a team.

### Overview information

**Main metrics of the gift cards stream**
- Providers initially reviewed and contacted: 20+
- Emails/chats initiated: 50+
- Providers thoroughly evaluated (account setup, demo calls, etc): 6
- Providers transferred for the Ubiquity testing: 1 (tremendous.com)

**Team expectations**
- The proposal to research Gift cards for payouts was presented on one of the research meetings.
The team reaction was like indifferent-negative.
- It was resulted in not informing the team about the progress on this stream to avoid non-relevant questions.

**Current status** 
- Waiting for the testing results of 1 provider
- https://github.com/ubiquity/research/issues/71

- I'm looking for a solution that provides more customizable cards so that they are perceived as "branded by Ubiquity".

Currently, there are negotiations with 
https://www.tangocard.com/reward-catalog?rewardcategory=cash+equivalent

https://huuray.com/products/create-your-own-gift-card/ 

**Complexity of the card issuance for crypto companies**

Even top crypto companies with large userbase and excellent legal support can face significant issues with card issuing.
A recent example is Binance
https://www.bloomberg.com/news/articles/2023-08-24/mastercard-to-end-its-binance-co-branded-card-partnership

Ubiquity is considered as much more risky partner than Binance by all criteria.
This makes the Ubiquity card issuing highly complex.

We should form realistic expectation for the card issuance and be flexible with our requirements.

**Risks and potential issues**

- According to providers policies, gift cards should be used appropriately otherwise our account can be banned.
The violation can include distribution for sitizens of sanctioned jurisdictions (we do not track this but when a purchase is performed it can be tracked), massive or too regular payouts (can be assesed as a money laundering for own needs), relation to crypto (due to lack of regulations), etc

- This means that we should have at least 2 providers setup and running.
We can use them for prototyping but not rely on them as a stable flow for all fiat cash-out payouts.

**Comment by 0x4007** [comment]
Thanks for the update. 

After looking through the `tremendous` web app (it's pretty polished) I think we can leverage the gift cards and clearly brand it as an "early prototype" of what will eventually be our fiat cash-out infra. 

I still have to top up the balance to fully test the flow. 

---

As a side note, I see it possible that research can be "bountified" just as long as a report on your findings is delivered (per my usual request, as a markdown document in a pull request.) But this might only be useful if you have multiple researchers working on the same initiative. Otherwise at this stage, it might just be useless overhead to share the minutiae of research on the card issuance initiative.

**Comment by sergfeldman** [comment]
@pavlovcik @hodlatoor Tango is one of the largest providers that has better choice of prepaid cards and potentially more customisation than Tremendous.
Also, due to the risks that I added to the issue body ^ we need at least 2 providers setup.

Can you please review Tango TOS before we sign the general agreement (via DocuSign)?
https://www.tangocard.com/legal/etos#SummaryETOS

**Comment by 0x4007** [comment]
How can I sign on docusign? I tried registering but realized its a process for international companies. I was directed to https://www.tangocard.com/intnl-form which seems to require manual review. Perhaps you have an account already?

**Comment by sergfeldman** [comment]
> How can I sign on docusign? I tried registering but realized its a process for international companies. I was directed to https://www.tangocard.com/intnl-form which seems to require manual review. Perhaps you have an account already?

@pavlovcik Yes, I have the account. After the manual review they should send an email with documentation to sign.
Are you ok with their TOS?

**Comment by sergfeldman** [comment]
> @pavlovcik @Hodlatoor Tango is one of the largest providers that has better choice of prepaid cards and potentially more customisation than Tremendous. Also, due to the risks that I added to the issue body ^ we need at least 2 providers setup.
> 
> Can you please review Tango TOS before we sign the general agreement (via DocuSign)? https://www.tangocard.com/legal/etos#SummaryETOS

@pavlovcik @Hodlatoor please clarify when to expect on your review of the Tango TOS

**Comment by Draeieg** [comment]
> Ubiquity is considered as much more risky partner than Binance by all criteria.

I mean we're not Sued by the SEC (yet) and haven't been involved in money laundering rumors lol

**Comment by 0x4007** [comment]
> to expect on your review of the Tango TOS

I think for all of the card issuance initiatives we can just set them up and retroactively check tos etc. 

1. I assume they have reasonable terms
2. We can always switch to another provider if they have something crazy
